docs/references/go/verifying-sessions.mdx

```diff
@@ -128,20 +121,85 @@ func protectedRoute(jwks *clerk.JSONWebKeySet) func(http.ResponseWriter, *http.R
       w.Write([]byte(`{"access": "unauthorized"}`))
       return
     }
+    fmt.Fprintf(w, `{"user_id": "%s"}`, claims.Subject)
+  }
+}
+```
+</InjectKeys>
 
-    // Fetch the correct JSON Web Key
-    var jwk *clerk.JSONWebKey
-    for _, k := range jwks.Keys {
-      if k.KeyID == unsafeClaims.KeyID {
-        jwk = k
-        break
-      }
-    }
+If you need to verify session tokens frequently, it might be a good idea to cache the JSON Web Key for your instance, to avoid making too many requests to the Clerk Backend API.
+
+The following snippet repeats the same example as above. However, it demonstrates a possible way to store your JSON Web Key. The example is meant as a rough guide, implementation may vary depending on your needs.
```

Comment:
```suggestion
If you need to verify session tokens frequently, it's recommended to cache the JSON Web Key for your instance in order to avoid making too many requests to the Clerk Backend API.

The following example includes the same code as above, with the addition of demonstrating a possible way to store your JSON Web Key. The example is meant to serve as a guide; implementation may vary depending on your needs.
```